pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-sytem]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.isort]
+profile = "black"
+skip_gitignore = true
+# known_first_party=["great_expectations","importlib_metadata"] # Added here because seed-isort-config does not recognize it
```

Comment:
Need to verify that this still works OUTSIDE of a virtual environment.
I know isort made some improvements to the way it detects these packages recently, but some of this may still be needed.